src/engine/framework/System.cpp

```diff
@@ -124,10 +124,10 @@ static void CreateSingletonSocket()
 	addr.sun_family = AF_UNIX;
 	Q_strncpyz(addr.sun_path, singletonSocketPath.c_str(), sizeof(addr.sun_path));
 	if (bind(singletonSocket, reinterpret_cast<struct sockaddr*>(&addr), sizeof(addr)) == -1)
-		Sys::Error("Could not bind singleton socket: %s", strerror(errno));
+		Sys::Error("Could not bind singleton socket at file: \"%s\", error: \"%s\"", singletonSocketPath.c_str(), strerror(errno) );
```

Comment:
Remove the `.c_str()`. Otherwise LGTM

Reply:
Oh, it cab take strings? I'll be happy to oblige!